MLIR.cmake

```diff
@@ -63,12 +63,12 @@ function(add_onnx_mlir_dialect_doc dialect dialect_tablegen_file)
           OUTPUT ${GEN_DOC_FILE}
```

Comment:
This needs to change too now because the output has changed.

Reply:
I would recommend just changing GEN_DOC_FILE to point to the source rather than the bin directory if that's the goal then the rest of the function does not need to change.

Reply:
Thanks, that makes a lot of sense. I am really not an expert at cmake files, learning every days.